src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURE_AMOUNT = 10;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] array = new Figure[FIGURE_AMOUNT];
```

Comment:
Let's give this array a more personal name. I think it is not just any array, but the Array fo Figures. Don't make him offended)